nrf_modem/doc/CHANGELOG.rst

```diff
@@ -9,6 +9,18 @@ Changelog
 
 All notable changes to this project are documented in this file.
 
+nrf_modem 1.0.2
+***************
+
+* Implement RAI (Release Assistance Indication) support in modem lib.
+* Fixed issue with socket state machine so that a RX event is not triggered on a socket closed by peer. Which would trigger both a POLLIN and a POLLHUP on socket closed by peer.
+* Fixed non-blocking recv behavior when WAITALL or DONTWAIT flags was used.
+* Fixed the behavior for blocking recv calls when not all data was received immediately.
+* Optimized memory usage when IPv4 addresses are used.
```

Comment:
`Reduced Heap memory usage in recv() by 20% when using IPv4`

Reply:
ok